packages/otel/src/constants.ts

```diff
@@ -0,0 +1,2 @@
+export const GET_TRACER = Symbol('getTracer')
```

Comment:
Do we not need `Symbol.for` here? If I understand correctly, we'll have two instances of `@netlify/otel`: one inline into `serverless-functions-api` and the other one loaded as a dependency from user code. I think `Symbol("getTracer")` will map to two different symbols, whereas `Symbol.for("getTracer")` would point to the same one?

Reply:
Yes that's true, perhaps we should just use a string instead of adding to the global symbol registry

Reply:
Works for me!

Reply:
Done